### DISCUSSION TITLE
Installing on home assistant 10

### DISCUSSION THREAD

**Discussion by kikili** [tsightler/ring-mqtt#614] [Support Q&A] [answered]
I just acquire a Ring Alarm and try to integrate it with home assistant (ras, but when I install the add-on I get this error message:
23-04-21 09:02:13 INFO (MainThread) [supervisor.addons] Creating Home Assistant add-on data folder /data/addons/data/5e5dfff6_ring_alarm
23-04-21 09:02:13 ERROR (SyncWorker_7) [supervisor.docker.addon] Invalid build environment, can't build this add-on!

On raspberry pi4
Home Assistant 2023.4.5
Supervisor 2023.04.0
Operating System 10.0
Interface utilisateur : 20230411.1 - latest

**Comment by tsightler** [comment, marked as answer]
> 5e5dfff6_ring_alarm

I'm not sure exactly what instructions you are attempting to follow, but that is not this project, but rather looks like some cloned repository of this project from many years ago.  If you want to use this project, please follow the instructions on the [official ring-mqtt addon](https://github.com/tsightler/ring-mqtt-ha-addon) page.

**Comment by kikili** [comment]
Yes very sorry.
Thanks for the link
